src/System.Management.Automation/engine/hostifaces/RunspacePoolInternal.cs

```diff
@@ -84,6 +84,7 @@ internal class RunspacePoolInternal
             pool = new Stack<Runspace>();
             runspaceRequestQueue = new Queue<GetRunspaceAsyncResult>();
             ultimateRequestQueue = new Queue<GetRunspaceAsyncResult>();
+            _initialSessionState = InitialSessionState.CreateDefault();
```

Comment:
Are we sure that we want to use CreateDefault() and load all commands/functions?  Or CreateDefault2() where just Core commands are loaded (and is faster)?  Please add a comment to justify which default we use.

Reply:
Based on Jason's test, CreateDefault() is the faster one while CreateDefault2() does more, I'll update the comments on those methods

Reply:
Note that it feels like something is terribly wrong with `CreateDefault2`, so that's investigating. It should be faster like Paul suggested.

Reply:
I think that should be a separate issue.  https://github.com/PowerShell/PowerShell/issues/5104